pkg/controller/util_test.go

```diff
@@ -394,6 +608,75 @@ func createCDIConfigWithStorageClass(name string, storageClass string) *cdiv1.CD
 	}
 }
 
+func createCDIConfigWithStorageClassPreallocation(storageClass string, preallocation bool) *cdiv1.CDIConfig {
+	return &cdiv1.CDIConfig{
```

Comment:
What do you think about having a default empty CDI, and just adding &cdiv1.Preallocation{} part in the following 3 functions